tests/integration/events/scraper-test.js

```diff
@@ -0,0 +1,68 @@
+process.env.NODE_ENV = 'testing'
+
+const sandbox = require('@architect/sandbox')
+const test = require('tape')
+const { join } = require('path')
+const intDir = join(process.cwd(), 'tests', 'integration')
+const testCache = require(join(intDir, '_lib', 'testcache.js'))
+const fakeCrawlSites = require(join(intDir, '_lib', 'fake-crawl-sites.js'))
+const crawlerHandler = require(join(process.cwd(), 'src', 'events', 'crawler', 'index.js')).handler
+const scraperHandler = require(join(process.cwd(), 'src', 'events', 'scraper', 'index.js')).handler
+
+/** Create AWS event payload for the crawl/scrape handlers. */
+function makeEventMessage (hsh) {
+  return { Records: [ { Sns: { Message: JSON.stringify(hsh) } } ] }
+}
+
+test('scrape extracts data from cached file', async t => {
+  await sandbox.start({ port: 5555, quiet: true })
+
+  t.plan(9)
+
+  const sourcesPath = join(intDir, 'fake-sources')
+
+  const rawData = { cases: 10, deaths: 20 }
+  fakeCrawlSites.writeFile('fake', 'fake.json', JSON.stringify(rawData))
+
+  testCache.setup()
+  const event = makeEventMessage({ source: 'fake', _sourcesPath: sourcesPath })
+  await crawlerHandler(event)
+  t.equal(1, testCache.allFiles().length, 'Sanity check, have data to scrape.')
+
+  const fullResult = await scraperHandler(event)
+  const result = fullResult[0]
+  t.ok(result, 'Have result')
+
+  t.equal('iso1:us#iso2:us-ca', result.locationIDs.join(), 'Location IDs')
+
+  const actual = result.data
+  t.ok(actual, 'Have data')
+  t.equal(1, actual.length, '1 record in returned data')
+
+  // These fields should match exactly.
+  const expected = {
+    cases: 10,
+    deaths: 20,
```

Comment:
Should probably add more fields, max out the expected output so we can make sure *nothing* gets lost along the way.

Reply:
I'm not sure what fields we can include, so this is a first iteration. :-)

Reply:
Let's add a TODO for this one.

Reply:
TODO added.